src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -39,9 +39,9 @@ public class AzureAppConfigurationOptions
         private SortedSet<string> _keyPrefixes = new SortedSet<string>(Comparer<string>.Create((k1, k2) => -string.Compare(k1, k2, StringComparison.OrdinalIgnoreCase)));
 
         /// <summary>
-        /// The connection string to use to connect to Azure App Configuration.
+        /// The list of connection strings used to connect to Azure App Configuration.
```

Comment:
Can we reword it so it's clear the connection strings must be for the same store?